cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/lib.rs

```diff
@@ -835,7 +835,7 @@ impl_runtime_apis! {
 			PolkadotXcm::query_xcm_weight(message)
 		}
 
-		fn query_delivery_fees(destination: VersionedLocation, message: VersionedXcm<()>) -> Result<VersionedAssets, XcmPaymentApiError> {
+		fn query_delivery_fees(destination: VersionedLocation, message: VersionedXcm<()>, _: VersionedAssetId) -> Result<VersionedAssets, XcmPaymentApiError> {
```

Comment:
Even though bridge hub doesn't support multiple assets for fee payment, I would still throw an error if the asset is not the expected one (`{ parents: 1, interior: Here }`)